src/zb_links/__init__.py

```diff
@@ -0,0 +1,34 @@
+import os
+
+from pkg_resources import get_distribution
+
+from zb_links.db.models import db
+
+
+def configure_app(flask_app):
+    default_config = {
+        "FLASK_APP": "zb_links.app.py",
+        # follow recommended settings to save memory
+        # see https://flask-sqlalchemy.palletsprojects.com/en/2.x/config/
+        "SQLALCHEMY_TRACK_MODIFICATIONS": False,
+        "SWAGGER_UI_DOC_EXPANSION": "list",
+        "RESTPLUS_VALIDATE": True,
+        "RESTPLUS_MASK_SWAGGER": False,
+        "ERROR_404_HELP": False,
+        "DEBUG": False,
+        "TESTING": False,
+        "CSRF_ENABLED": True,
+        "API_VERSION": get_distribution("zbmath_links_api").version,
```

Comment:
Can we use a fallback version here? It might be that get_distribution tries to use the version from scm which might not be available. So I think it would be a good idea to move the default out of this block and determine it before within a try catch block.

Reply:
A question. What error are we trying to catch when you say "determine it before within a try catch block"?

Reply:
I was thinking of something like this

https://pythonrepo.com/repo/pypa-setuptools_scm-python-build-tools#retrieving-package-version-at-runtime

maybe we can ultimately, remove the additional file
 https://github.com/zbMATHOpen/linksApi/blob/master/setup_docker.py
even though this issue might be unrelated.

Reply:
The other problem might be solved via https://github.com/pypa/setuptools_scm/issues/568. However, it still would not be able to determine the version at runtime.

Reply:
> I was thinking of something like this
> 
> https://pythonrepo.com/repo/pypa-setuptools_scm-python-build-tools#retrieving-package-version-at-runtime

Sorry, but from this link it is not clear the role of the "catch block" you mentioned. If a try catch block is needed, what error would arise which would be required to be caught?

Reply:
Maybe just leaf it as it is, and we fix the problem in another pull request